src/material/tooltip/tooltip.ts

```diff
@@ -791,6 +795,7 @@ export abstract class _TooltipComponentBase implements OnDestroy {
     this._showTimeoutId = setTimeout(() => {
       this._visibility = 'visible';
       this._showTimeoutId = undefined;
+      this._isMultiline = this._isTooltipMultiline();
```

Comment:
This logic is only relevant for the MDC tooltip so it should be moved into the MDC classes.